examples/dashboard/.saturn/saturn.json

```diff
@@ -0,0 +1,12 @@
+{
+    "image": "saturncloud/saturn:2021.02.22",
+    "jupyter": {
+        "size": "medium",
+        "disk_space": "4Gi",
```

Comment:
I missed this before, but this needs to be a valid `disk_space` option. Maybe you meant to put `10Gi` or `40Gi`?

Reply:
Why is 4 not valid? I wanted to conserve space since we shouldn't need much I don't think. Is there a minimum required?

Reply:
It's a selector, so there are only certain options. You can see this reflected in the UI.